lib/plugins/create/templates/aws-java-gradle/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'java'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile (
+        'com.amazonaws:aws-lambda-java-core:1.1.0',
+        'com.amazonaws:aws-lambda-java-events:1.1.0'
+    )
+}
+
+task buildZip(type: Zip) {
```

Comment:
Could you add a short comment describing that you're packaging all dependencies here in one zip file that can be deployed through the artefact configuration later on in the serverless.yml? Just so people know whats going on